akka-actor/src/main/scala/akka/util/PrettyDuration.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
  */
-package akka.testkit.metrics.reporter
+package akka.util
```

Comment:
It's Java API, can't have default arguments :-)
But yes, let's do an overload with `def format(duration) = format(duration, includeNanos = false, precision = 4)`